src/ui/index.js

```diff
@@ -52,6 +59,9 @@ export default async (kbnServer, server, config) => {
       const app = uiExports.apps.byId[id];
       if (!app) return reply(Boom.notFound('Unknown app ' + id));
 
+      const acceptLanguageStr = req.headers['accept-language'];
```

Comment:
@srl295 Good catch, thanks. I have updated the algorithm for deciding the  translation language to choose the default language if no user preferred language header passed.